docs/rules/prefer-replace-range.md

```diff
@@ -0,0 +1,52 @@
+# prefer using replaceText instead of replaceTextRange. (prefer-replace-text)
+
+## Rule Details
+
+The rule reports an error if `replaceTextRange`'s first argument is an array of identical array elements. It can be easily replaced by `replaceText` to improve readability.
+
+Examples of **incorrect** code for this rule:
+
+```js
+/* eslint eslint-plugin/prefer-text-range: error */
+module.exports = {
+  create(context) {
+    context.report({
+      fix(fixer) {
+        // error, can be writen: return fixer.replaceText([node, '');
```

Comment:
Nitpick: the word "written" is spelled incorrectly here. Also, the comment here is missing a `]`.